.github/workflows/build.yaml

```diff
@@ -77,7 +81,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v3
         with:
-          python-version: "3.10"
+          python-version: "3.9"
```

Comment:
Why is this now 3.9?

Reply:
3.9 and 3.10 pass the matrix test. Would we pick the lowest python version for publishing? Or just use 3.10 and 
also declare 3.10 required?

Reply:
Changed it back to 3.10